buildrpm/oci-utils.spec

```diff
@@ -84,6 +84,9 @@ Utilities unit tests
 %{__cp} -r setup.py %{buildroot}/opt/oci-utils
 %{__cp} -r requirements.txt %{buildroot}/opt/oci-utils
 %{__cp} -r README %{buildroot}/opt/oci-utils
+# temporary workaround to EOL vnci script: move it else where
+%{__mv} %{buildroot}/usr/libexec/secondary_vnic_all_configure.sh %{buildroot}%{python2_sitelib}/oci_utils/impl
```

Comment:
I would suggest we rename it as well so that it does not show up when someone do a find...

Reply:
renamed the file